### PR TITLE
8.0 account_netting add multi-currency support

### PR DESCRIPTION
**Pull request by luc-demeyer**
[8.0] add multi-currency support to account_netting

### CODE REVIEW

File: account_netting/__openerp__.py [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
+# (c) 2016 Noviat nv/sa (www.noviat.com)
 # License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
 
 {
     'name': 'Account netting',
-    'version': '8.0.1.0.0',
+    'version': '8.0.2.0.0',
     'summary': "Compensate AR/AP accounts from the same partner",
     'category': 'Accounting & Finance',
     'author': 'Serv. Tecnol. Avanzados - Pedro M. Baeza, '
```
Comment (pedrobaeza) [inline account_netting/__openerp__.py]: Can you please take the occasion to switch the authorship to my new company `Tecnativa`?

Comment (luc-demeyer) [reply account_netting/__openerp__.py]: Done.

**Comment by luc-demeyer** [comment]
@fclementic2c 
Frédéric,
since you living on a CHF island in the EUR zone, multi-currency is part of your daily business. Could you find a bit of time to review/test this multi-currency enhancements of the account_netting module ?

**Comment by gurneyalex** [comment]
@luc-demeyer can you rebase the PR and resolve the conflict?

**Comment by luc-demeyer** [comment]
@gurneyalex 
PR rebased hence can be merged now.
Travis fails on e.g. account_asset_management but to fix this PR  501 must me merged (and this PR really should be merged asap since it's a LOT better than what's currently published and I am running it already at 10+ customers hence it's stable).

**Pull request closed without merging**